cos_registration_server/api/dashboards.py

```diff
@@ -3,12 +3,13 @@
 from os import mkdir, remove
 from pathlib import Path
 from shutil import rmtree
+from typing import Any
 
 from devices.models import Device
 from django.conf import settings
 
 
-def add_dashboards(device):
+def add_dashboards(device: Any) -> None:
```

Comment:
What's the actual type expected here?

Reply:
The actual type is Device, but because it's from a django generator from the serializer, the type there is Any.
This is a limitation of django https://code.djangoproject.com/ticket/33174